emails/tests/models_tests.py

```diff
@@ -724,8 +724,12 @@ def test_when_premium_available_returns_True(self) -> None:
         self.set_fxa_locale("de-DE,en-xx;q=0.9,en;q=0.8")
         assert self.profile.fxa_locale_in_premium_country is True
 
-    def test_when_premium_unavailable_returns_False(self) -> None:
+    def test_en_implies_premium_available(self) -> None:
```

Comment:
This was a "wat" from https://github.com/mozilla/fx-private-relay/pull/3602#discussion_r1261725670 - @groovecoder expected `en` to imply `us`, and now it does, both in `Profile.fxa_locale_in_premium_country` and in a web request.